atlas/commands/bootstrap/templates/cmd/server/swagger.go.gotmpl

```diff
@@ -5,7 +5,8 @@ import "net/http"
 func NewSwaggerHandler(swaggerDir string) http.Handler {
 	mux := http.NewServeMux()
 	mux.HandleFunc("/swagger/", func(writer http.ResponseWriter, request *http.Request) {
-		http.ServeFile(writer, request, swaggerDir)
+		subpath := strings.SplitN(request.URL.Path, "/swagger/", 2)[1]
```

Comment:
You might consider using `http.StripPrefix(...)` to remove the leading `/swagger/` from a URL. Also, the `strings.TrimPrefix` could work well here. Just bringing this up because I'm deathly afraid of accessing arrays that are assumed to be non-empty.

https://golang.org/pkg/net/http/#StripPrefix
https://golang.org/pkg/strings/#TrimPrefix